mosip-identity-plugin/pom.xml

```diff
@@ -3,11 +3,17 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 
+	<parent>
+		<groupId>io.mosip.esignet.plugin</groupId>
+		<artifactId>esignet-plugin-parent</artifactId>
+		<version>1.0.0-SNAPSHOT</version>
+	</parent>
 
-	<version>1.2.1.0</version>
-	<artifactId>esignet-integration-impl</artifactId>
-	<name>esignet-integration-impl</name>
-    <description>e-Signet Integration Implementation Library</description>
+	
+	<version>1.0.0-SNAPSHOT</version>
```

Comment:
Any particular reason for defaulting to 1.0.0 version ?

Reply:
As both artifactId and groupId are changed, I think it makes sense to start with 1.0.0